tests/unit/test_repos_repo.py

```diff
@@ -1473,6 +1486,12 @@ def test_import_issue(self):
                                        body='Foobar body',
                                        created_at='2014-03-16T17:15:42Z')
```

Comment:
Take a look at this timestamp for your unit test above. It doesn't have to be this exact one, but it should be in this format.